src/styles/process.scss

```diff
@@ -0,0 +1,164 @@
+@import "vars";
+
+.process {
+
+  &__back {
+    width: 100%;
+    height: 816px;
+    background-color: $lightgray;
+    box-sizing: border-box;
+    padding-left: 163px;
+    display: flex;
+  }
+
+  &__nocard {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  &__h2 {
+    padding-top: 162px;
+  }
+
+  &__h3 {
+    margin-top: 19px;
+    max-width: 362px;
+  }
+
+  &__p {
+    max-width: 362px;
+    margin-top: 64px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 24px;
+    color: $darkblue;
+    mix-blend-mode: normal;
+  }
+
+  &__cards {
+    margin: 225px 0 0 187px;
+    display: flex;
+    min-width: 598px;
+    max-width: 598px;
```

Comment:
if you specify min-width and max-width, you will set up a lower and upper bound and if both are equal it will be the same as simply specifying a width.